### PR TITLE
Updating footer and removing advisors page

### PR DESCRIPTION
**Pull request by hannahbergam**
Removes the /advisors page and its one reference in the footer. This is a bit more complex of a translation. I'm writing a new string for translation (instead of updating the existing one) so that it isn't missed in the pipeline.

Before:
Screenshot 2024-01-26 at 4 21 40 PM

After (on dance party):
Screenshot 2024-02-06 at 2 52 31 PM

## Links

- spec: []()
- jira ticket: https://codedotorg.atlassian.net/browse/ACQ-1435

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/about/advisors.md.erb [hunk 1]
```diff
@@ -1,9 +0,0 @@
----
-title: Education Advisors
-nav: about_nav
-theme: responsive
----
-# Education Advisory Council
-Our Education Advisory Council is a group of educators with decades of experience in teaching Computer Science, across all grade levels - elementary, secondary, or at the university level.
-
-<%= view :about_advisors, people:DB[:cdo_leaders].where(kind_s:'education') %>
```
Comment (TurnerRiley) [inline pegasus/sites.v3/code.org/public/about/advisors.md.erb:9]: Can we remove `about_advisors` too?

**Review by TurnerRiley — APPROVED** [review]
Thanks for making it a new string! Just had one question

File: pegasus/sites.v3/code.org/public/about/advisors.md.erb (hunk 1 above)
Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/about/advisors.md.erb:9]: oooo good catch! I'll remove this once we can pull staging again:)

Comment (hannahbergam) [reply pegasus/sites.v3/code.org/public/about/advisors.md.erb:9]: Removed!

**Review by TurnerRiley — APPROVED** [review]

**Pull request merged**